### PR TITLE
bugfix `this.err` + new logo + minor refactor

### PR DESCRIPTION
**Pull request by ghost**
Works as expected, but let me know if this looks good to you.

Please enable tests on your Travis dashboard also :+1:

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,31 +1,26 @@
...
-    opts.icon = devicon(opts.icon)
-    try {
-      notifier.notify(opts, function (e) { if (e) { throw e } })
-    } catch (e) { throw e }
+  this.notify = function (options) {
+    notifier.notify(assign(options, {
+      title: "Fly",
+      message: "Task completed.",
+      icon: options.icon
+        ? devicon(options)
```
Comment (MadcapJake) [inline index.js]: Shouldn't this be `devicon(options.icon)`?

Comment (ghost) [inline index.js]: My bad. Will fix.

**Comment by MadcapJake** [comment]
Thanks! :+1:

**Comment by ghost** [comment]
:+1:

**Pull request merged**